phabricator/manifests/phabricator-rc.yaml

```diff
@@ -68,15 +68,19 @@ spec:
         - name: https
           containerPort: 443
         livenessProbe:
-          httpGet:
-            path: /
-            port: http
+          exec:
+            command:
+            - harpoon
+            - status
+            - phabricator
           initialDelaySeconds: 120
```

Comment:
Yes.. in my test it seems to be fine. 
Maybe we can raise it to 180 secs to be on the safer side